init.lua

```diff
@@ -146,13 +146,12 @@ lightning.strike = function(pos)
 
 		-- only affect players inside effect_range
 		if distance < lightning.effect_range then
-			local sky = {}
-			sky.bgcolor, sky.type, sky.textures = player:get_sky()
```

Comment:
This code was used to preserve changes made by other mods. By overwriting the entire sky table, you will undo their changes and hence break compatibility.